pkg/installer/installer_test.go

```diff
@@ -86,12 +86,14 @@ var _ = Describe("Installer", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(repo.GetName()).To(Equal("test"))
 			Expect(helpers.Exists(spec.Rel("repository.yaml"))).ToNot(BeTrue())
-			Expect(helpers.Exists(spec.Rel(TREE_TARBALL))).ToNot(BeTrue())
+			Expect(helpers.Exists(spec.Rel(TREE_TARBALL + ".gz"))).ToNot(BeTrue())
```

Comment:
Does it means we don't test anymore the case when it is *not* compressed?

Reply:
Yup, we need to add a test for this :D

Reply:
Done